### PR TITLE
model training notebook, saved model

### PR DESCRIPTION
**Pull request by ezzaldeeen**
- Model training
- Hyperparameter Tuning
- Regularization
- Saved model

#### Hyperparameters:
-  units: 384
- dropout: False
- learning_rate: 0.0005
- Score: 97%

### CODE REVIEW

**Comment by obadakhalili** [comment]
Great work, teammate
